api/core/helper/code_executor/code_executor.py

```diff
@@ -85,4 +75,32 @@ def execute_code(cls, language: Literal['python3', 'javascript', 'jinja2'], code
         if response.data.error:
             raise CodeExecutionException(response.data.error)
         
-        return template_transformer.transform_response(response.data.stdout)
\ No newline at end of file
+        return response.data.stdout
+
+    @classmethod
+    def execute_workflow_code_template(cls, language: Literal['python3', 'javascript', 'jinja2'], code: str, inputs: dict) -> dict:
+        """
+        Execute code
+        :param language: code language
+        :param code: code
+        :param inputs: inputs
+        :return:
+        """
+        template_transformer = None
+        if language == 'python3':
```

Comment:
nit: is `match-case` a better option for handling conditions without repeating the object?
https://docs.python.org/3.10/tutorial/controlflow.html#match-statements